src/fprime_bootstrap/__main__.py

```diff
@@ -45,13 +47,47 @@ def main():
         help="Version of F´ to checkout (default: latest release)",
     )
 
+    clone_parser = subparsers.add_parser(
+        "clone", help="Clone an existing remote F´ project"
+    )
+    clone_parser.add_argument(
+        dest="url",
+        type=str,
+        help="URL of the remote repository to clone",
+    )
+    clone_parser.add_argument(
+        "--path",
+        type=str,
+        help="Path to create the project in (default: current directory)",
+        default=os.getcwd(),
+    )
+    clone_parser.add_argument(
+        "--no-venv",
+        action="store_true",
+        help="Do not create a virtual environment in the project",
+        default=False,
+    )
+    clone_parser.add_argument(
+        "--fprime-subpath",
+        type=str,
+        help="Relative path from root of project to F´ submodule (default: ./fprime)",
+        default="./fprime",
+    )
+    clone_parser.add_argument(
```

Comment:
Does this not come it with the above path?  Like couldn't the path rename too?

Reply:
We could have it do that. Currently `--path` specifies what the parent directory of the project root is. Basically equivalent to
```
cd <path>
git clone https://project.url '<rename>'
```

The reason for this is I wanted the `fprime-bootstrap clone` to feel similar to `git clone` but still be consistent with our other toolings (e.g. `fprime-util build --path` where path is where to execute the command from).

Reply:
So the simple use case is:

```
cd <path>
fprime-bootstrap clone my-project-clone
```

Makes sense

Reply:
Yes! I listed some example usages in the description of the PR